sdks/v4-sdk/src/PositionManager.test.ts

```diff
@@ -367,6 +367,45 @@ describe('PositionManager', () => {
       expect(value).toEqual('0x00')
     })
 
+    it('succeeds when migrating out of range and need to transfer eth', () => {
```

Comment:
would love a test for non-eth too, and for ETH-WETH wrapping combos if they are to be supported too